dv/uvm/core_ibex/scripts/run-iss.py

```diff
@@ -45,7 +45,8 @@ def main() -> int:
     cmd = [spike, '--log-commits', '--isa', iss_isa, '-l', args.input]
     return run_one(args.verbose,
                    cmd,
-                   redirect_stdstreams=args.output)
+                   redirect_stdstreams=args.output,
+                   timeout_s=30)  # Spike can run indefinitely in some cases
```

Comment:
It's usually pretty quick for running standalone. 30s is usually safe. But this will get ripped out when all the non-cosim stuff is removed so it won't matter massively.